src/keria/app/delegating.py

```diff
@@ -226,23 +227,23 @@ def on_post(self, req, rep, name):
         except (kering.AuthError, ValueError) as e:
             raise falcon.HTTPBadRequest(description=e.args[0])
     
-    @staticmethod
-    def approveDelegation(agent, hab, anc) -> str:
-        serder = serdering.SerderKERI(sad=anc)
+@staticmethod
```

Comment:
If this method is no longer a part of the class, this annotation is inappropriate